packages/ssf-cli/index.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+/* ssf.js (C) 2013-present SheetJS -- http://sheetjs.com */
+/* eslint-env node */
+/* eslint no-console:0 */
+/*jshint node:true */
+var X = require('ssf');
+function run() {
+  var argv = process.argv.slice(2);
+  if (argv.length < 2 || argv[0] == "-h" || argv[0] == "--help") {
```

Comment:
Sometimes I write this like:
```js
if(['-h', '--help'].indexOf(argv[0]) > -1)
```

But that's usually with more than two options. It's totally fine as is.